package.json

```diff
@@ -55,7 +55,11 @@
     "winston": "^2.2.0",
     "ws": "^1.0.1",
     "xss": "^0.2.10",
-    "yargs": "^4.2.0"
+    "yargs": "^4.2.0",
+    "chai": "3.5.0",
+    "eslint-plugin-jsx-a11y": "^1.0.2",
+    "eslint-plugin-react": "^5.0.1",
+    "auto-launch": "^2.0.1"
```

Comment:
Please move this to the top of this object.  It is good practice for the dependencies to be sorted by name